cabal-testsuite/PackageTests/ExtraPackages/cabal.out

```diff
@@ -0,0 +1,12 @@
+# cabal v2-update
+Downloading the latest package list from test-local-repo
+# cabal v2-run
+Resolving dependencies...
+Build profile: -w ghc-<GHCVER> -O1
+In order, the following will be built:
+ - some-exe-0.0.1.0 (exe:some-exe) (requires build)
+Configuring some-exe-0.0.1.0...
+Preprocessing executable 'some-exe' for some-exe-0.0.1.0..
+Building executable 'some-exe' for some-exe-0.0.1.0..
+Installing executable some-exe in <PATH>
+Warning: The directory <ROOT>/cabal.dist/home/.cabal/store/ghc-<GHCVER>/incoming/new-52499<ROOT>/cabal.dist/home/.cabal/store/ghc-<GHCVER>/sm-x-0.0.1.0-b845c0b5/bin is not in the system search path.
```

Comment:
Arggl this line is failing the test because of how package directories are being shortened on OSX vs. Linux. @phadej any idea where this monstrous path is coming from in the first place?

Reply:
Ah, I guess it's this note in ProjectBuilding.hs

```
            -- Note that the copy command has put the files into
            -- @$tmpDir/$prefix@ so we need to return this dir so
            -- the store knows which dir will be the final store entry.
```

Reply:
From `CmdInstall`, we warn if people try to install stuff somewhere which is not in PATH. That is reasonable.

In tests we obviously don't use global paths.

The current diff though says you are using `v2-run` so I'm confused. (force pushes confuse me, better to just push WIP commits until the final squash).

Reply:
I guess that's how extra-packages work: You tell cabal-install to run an exe from an extra-package and it will make sure that package is being build and installed before running.